lib/galaxy/tools/deps/container_resolvers/mulled.py

```diff
@@ -307,7 +312,7 @@ def __str__(self):
 
 @six.python_2_unicode_compatible
 class BuildMulledDockerContainerResolver(ContainerResolver):
-    """Look for mulled images matching tool dependencies."""
+    """Build for Docker mulled images matching tool dependencies."""
 
     resolver_type = "build_mulled"
```

Comment:
I don't feel strongly on this - I'd leave it as it is because it is easier and backward compatible - but if you'd like me to make it more explicit I certainly can.